pandas/io/json/normalize.py

```diff
@@ -241,6 +251,12 @@ def _recursive_extract(data, path, seen_meta, level=0):
         else:
             for obj in data:
                 recs = _pull_field(obj, path[0])
+                if recs == {}:
```

Comment:
Same comment around missing key - need to be cognizant of the `errors` parameter

Reply:
Not sure I understand. Setting `recs ` to an empty dictionary if there is a missing key (on line 199), is a convenient flag to then load the output with NaNs on line 258. How does the `errors `parameter come into this ?